### PR TITLE
Added file.md to common

### PR DESCRIPTION
**Pull request by dylanrees**

### CODE REVIEW

File: pages/common/file.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+> Determine file type
+
+- Give a description of the type of the specified file
+
+`file {{readme}}`
+`file {{python3.so.1}}`
+
+- Look inside a zipped file and determine the file type(s) inside
+
+`file -z foo.zip`
```
Comment (waldyrious) [inline pages/common/file.md]: Please use the `{{token syntax}}` for foo.zip

File: pages/common/file.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
+# file
+
+> Determine file type
+
+- Give a description of the type of the specified file
+
+`file {{readme}}`
+`file {{python3.so.1}}`
```
Comment (waldyrious) [inline pages/common/file.md]: One example should suffice, and preferably using a generic parameter name such as `{{filename}}`. I get that you're trying to make the point that the extension is unnecessary, but maybe it's better to say it explicitly in the example description.

**Comment by waldyrious** [comment]
Thank you! I added a couple inline comments.

**Comment by dylanrees** [comment]
Updated based on your comments.

File: pages/common/file.md [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+
+`file {{filename}}`
+
+- Look inside a zipped file and determine the file type(s) inside
+
+`file -z {{foo.zip}}`
+
+- Display a quick guide to various flags that can be used
+
+`file --help`
```
Comment (ostera) [inline pages/common/file.md]: I think this is unnecessary.

**Comment by ostera** [comment]
Other than the comment I left, looks good! :+1:

File: pages/common/file.md (hunk 3 above)
Comment (waldyrious) [inline pages/common/file.md]: Agreed.

**Comment by dylanrees** [comment]
Updated.

**Comment by waldyrious** [comment]
Great, I'll merge now. Thanks for your contribution!

**Pull request merged**